core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -740,6 +784,62 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
                     this.host
                 );
             },
+
+            [CUSTOMER_ACTIONS.COPY_CONTENTLET_INLINE_EDITING]: () => {
+                this.store.setEditorMode(EDITOR_MODE.EDIT);
+                const payload = <{ dataset: InlineEditingContentletDataset }>data.payload;
+
+                this.dotCopyContentModalService
+                    .open()
+                    .pipe(
+                        switchMap(({ shouldCopy }) => {
+                            if (!shouldCopy) {
+                                return of(payload);
+                            }
+
+                            return this.handleCopyContent();
+                        })
+                    )
+                    .subscribe((res: DotCMSContentlet) => {
```

Comment:
This can be either a `DotCMSContentlet` or `payload` type

Reply:
Updated! Now is `DotCMSContentlet` or `null`
Added both types and validation